### ISSUE TITLE
f01f9a81143e4b01e5ae6cdbb3a7350fff2f9919 is breaking change

### ISSUE THREAD

**Issue by froatsnook** [sunaku/vim-unbundle#10]
Hi,

Thank you very much for this plugin.  I've been using it for a couple years now and it works great.

Today I updated my vim plugins and suddenly nothing loaded!  I eventually figured out that 

```
:runtime vim-unbundle/unbundle.vim
```

wasn't working anymore because it changed to:

```
:runtime vim-unbundle/plugin/unbundle.vim
```

Could you put a big note in the README about this breaking change to help other people who stumble here looking for a solution?  IMO, it's easy to miss that one word in the command.

**Comment by sunaku** [comment]
Sorry about that. :sweat_smile: Now fixed in 870e65d390e0689e91ed70f73d7d6aa20649d0a7, thanks! :+1:

**Comment by froatsnook** [comment]
Ah, that's even better.  Thank you!
